src/java/org/apache/cassandra/db/streaming/CassandraStreamReceiver.java

```diff
@@ -256,6 +266,11 @@ public void finished()
 
                 // add sstables (this will build non-SSTable-attached secondary indexes too, see CASSANDRA-10130)
                 logger.debug("[Stream #{}] Received {} sstables from {} ({})", session.planId(), readers.size(), session.peer, readers);
+
+                // Don't mark as live until activated by the stream coordinator
+                if (session.streamOperation() == StreamOperation.TRACKED_TRANSFER)
+                    return;
+
                 cfs.addSSTables(readers);
 
                 //invalidate row and counter cache
```

Comment:
It makes sense that we wouldn't invalidate the caches here on a tracked import, but do we need to make sure we do it on activation?